doc/man/man3/crypto_lock.3monocypher

```diff
@@ -249,3 +249,13 @@ These functions implement RFC 8439, with XChacha20 instead of Chacha20.
 XChacha20 derives from Chacha20 the same way XSalsa20 derives from
 Salsa20, and benefits from the same security reduction (proven secure
 as long as Chacha20 itself is secure).
+.Sh HISTORY
+The
+.Fn crypto_lock
+and
+.Fn crypto_unlock
+functions first appeared in Monocypher 0.1.
+.Fn crypto_lock_aead
+and
+.Fn crypto_unlock_aead
+were introduced in Monocypher 1.1.0.
```

Comment:
Perhaps mention that a breaking change was introduced in version 2.0.0?

Reply:
I actually forgot about commit 09de649d and #89. Oops. Just shows how long ago we should've started with these sections.